Source/utils/sdl2_to_1_2_backports.cpp

```diff
@@ -897,7 +897,7 @@ char *SDL_GetPrefPath(const char *org, const char *app)
 			SDL_SetError("neither XDG_DATA_HOME nor HOME environment is set");
 			return NULL;
 		}
-#if defined(__unix__) || defined(__unix)
+#if defined(__unix__) || defined(__unix) || defined(__APPLE__)
 		append = "/.local/share/";

```

Comment:
This isn't the right path for Mac, on phone ATM but check the config Wiki for an example of the Path

Reply:
~/Library/Application Support/diasurgical/devilution

Reply:
Yep, this is now fixed in the latest version